components/inlineaiprompt/overview.md

```diff
@@ -0,0 +1,261 @@
+    };
+
+    private async Task ShowPromptAsync(MouseEventArgs e, string product)
+    {
+        PromptContext = product;
+        await InlinePromptRef.ShowAsync(e.ClientX, e.ClientY);
+    }
+
+    private void OnPromptRequest(InlineAIPromptPromptRequestEventArgs args)
+    {
+        args.Output = $"AI assistance for: {PromptContext}";
+    }
+}
+
+<style>
+    .products-container {
+        display: flex;
+        gap: 1rem;
+        flex-wrap: wrap;
+        padding: 1rem;
+    }
+
+    .product-card {
+        padding: 1rem;
+        background: #f9f9f9;
+        border: 1px solid #ddd;
+        border-radius: 8px;
+        min-width: 220px;
+        text-align: center;
+        cursor: pointer;
+    }
+</style>
+````
+
+## Streaming
+
+The InlineAIPrompt component supports streaming responses, letting users view AI-generated content in real time as it’s created. [Read more about the Blazor InlineAIPrompt streaming...](slug:inlineaiprompt-streaming)
+
+## Events
+
+The InlineAIPrompt component offers several events that allow developers to handle user interactions and processes effectively. [Read more about the Blazor InlineAIPrompt events...](slug:inlineaiprompt-events)
+
+## InlineAIPrompt Parameters
+
+The following table lists the InlineAIPrompt parameters. Also check the [InlineAIPrompt API Reference](slug:Telerik.Blazor.Components.TelerikInlineAIPrompt) for a full list of all properties, methods and events.
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `Class` | `string` | `The class that is rendered on the .k-prompt element.` |
+| `Commands` | `List<AIPromptCommandDescriptor>` | `Configures predefined commands for the component.` |
+| `EnableSpeechToText` | `bool` | `Specifies whether to enable speech to text functionality.` |
+| `OutputActions` | `List<AIPromptOutputActionDescriptor>` | `Defines the output actions.` |
+| `Placeholder` | `string` | `Placeholder text for the input field.` |
+| `Prompt` | `string` | `The value of the text field and the prompt that is sent to the AI service.` |
+| `PromptContext` | `string` | `Defines the additional context that is passed with the input/command prompt and system prompt.` |
+| `SystemPrompt` | `string` | `Defines the system prompt that is used when the component is integrated with MEAI.` |
+
+### Settings and Commands
+
+The InlineAIPrompt exposes settings for itself and its speech to text button. To configure the options, declare a  `<InlineAIPromptPopupSettings>` or `InlineAIPromptSpeechToTextButtonSettings` tag inside a `<InlineAIPromptSettings>` tag.
+
+The InlineAIPrompt component also exposes an option to set predefined commands. To configure the actions, use the `Commands` parameter and subscribe to the `OnCommandExecute` event that will fire whenever the user executes a command. The handler expects an argument of type `InlineAIPromptCommandExecuteEventArgs`.
```

Comment:
Explain what a command is, e.g. a predefined prompt that is processed immediately (if this is correct).